### DISCUSSION TITLE
HTML output

### DISCUSSION THREAD

**Discussion by gokulmaxi** [xournalpp/xournalpp#3797] [Q&A] [answered]
Hey, just wondering if there is any way to export the notes as HTML. I would like to export as HTML and host it in GitHub, So I can get a quick peek on my notes.

**Comment by Technius** [comment, marked as answer]
I'm not sure how direct conversion to HTML would work, given that the journal format is essentially a collection of images.

If you're dead set on using GitHub, one alternative might be to export your notes to PDF format and upload them to a repository. You can then build a webpage that uses a library like [PDF.js](https://mozilla.github.io/pdf.js/).

**Reply by gokulmaxi** [reply]
❤️
